lms/djangoapps/homepage/custom_context_processors.py

```diff
@@ -1,31 +1,13 @@
 from edx_notifications.server.web.utils import get_notifications_widget_context
 
+from .constants import CONFIGS
 
-def notifications_configs(request):
+
+def notifications_context(request):
```

Comment:
@mabdullahz are we sure that it won't have any impact on the caller of the function? The function is used in lms.envs.common.py as

CONTEXT_PROCESSORS = [
    'lms.djangoapps.homepage.custom_context_processors.notifications_context',
...]

Reply:
Avoid this suggestion because that would go outside the scope of this ticket. I still don't understand why we went ahead and refactored the function at all in this ticket since that was not part of the acceptance. We should have clearly defined the increased scope first, before touching anything other than the unit tests.

Reply:
Actually the configs were moved to a constant because they were also to be accessed in the unit test method, and I wanted a central location for them to be accessed. Regarding refactoring the name of the function, it was done at the suggestion of @zeshanarifios.